README.md

```diff
@@ -1,2 +1,36 @@
 # scs-pki-research
```

Comment:
An empty line is needed after the heading and a period at the end of the sentence.
Please check other headings, it is a good rule to wrap a heading with lines: two newlines before and one after.